library/src/main/java/com/jmolsmobile/landscapevideocapture/configuration/CaptureConfiguration.java

```diff
@@ -155,6 +158,20 @@ public boolean getAllowFrontFacingCamera() {
         return allowFrontFacingCamera;
     }
 
+    /**
+     * @return If front flash toggle must be displayed before capturing video
+     */
+    public boolean getAllowFlashToggle() {
+        return allowFlashToggle;
+    }
+
+    /**
+     * @return If front flash must be start on
```

Comment:
Does "front" need to be here? Flash also works for back camera, right?

Reply:
...voiceless...